### PR TITLE
Add language around default branch name

### PR DESCRIPTION
**Pull request by kmahar**
Open to any suggestions for wording improvements. Since this relates to version control specifically it seemed like a good fit as a sub-bullet but happy to move it to another section as well.

### CODE REVIEW

File: process/incubation.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ Changes to the Swift Server Ecosystem index page will be announced by the SSWG u
 * General
   * Has relevance to Swift on Server specifically
   * Publicly accessible source managed by an SCM such as github.com or similar
+    * Prefer to use `main` as the default branch name, in line with [Swift itself](https://forums.swift.org/t/moving-default-branch-to-main/38515) and all SSWG-owned repos
```
Comment (tomerd) [inline process/incubation.md]: indent intentional?

**Review by tomerd — APPROVED** [review]
lgtm

File: process/incubation.md (hunk 1 above)
Comment (kmahar) [reply process/incubation.md]: since it's related to SCM use I thought it made sense here

**Comment by kmahar** [comment]
Should I merge this or do we want any more members to review first?

**Comment by tomerd** [comment]
@kmahar changes requires supermajority vote: https://github.com/swift-server/sswg/blob/main/process/incubation.md#change-management

**Comment by tomerd** [comment]
cc @swift-server/sswg

File: process/incubation.md (hunk 1 above)
Comment (ktoso) [reply process/incubation.md]: The phrasing of SSWG-owned sounds a bit weird to me, seems like a new term? 

I know what you mean and it's kind of right but the phrasing feels a bit off: Is it really that SSWG "owns" repos? Maybe we can use stronger language and only refer to the swift guidelines to emphasise the point?

I would not want to create the impression that in order to "be a valuable project in SSWG ecosystem I somehow have to let go of ownership" etc, which sometimes I feel it seems like to people, while we're just a collection of projects, owned by various people, who try to keep a same style and standard of quality etc.

```suggestion
    * Use `main` as the default branch name, in line with [Swift's guidelines](https://forums.swift.org/t/moving-default-branch-to-main/38515)
```

wdyt?

**Review by ktoso — APPROVED** [review]
LGTM, wondering about specific wording but not a blocker 👍

File: process/incubation.md (hunk 1 above)
Comment (kmahar) [reply process/incubation.md]: sgtm to remove that wording, was mainly just trying to communicate that SSWG has/will do this for all of the repos under the `swift-server` org - but I think Swift's guidelines are enough to reference on their own.

there was some some discussion around whether we should _require_ this, and if so what impact that would have on existing repos in the incubation process, and I think the consensus was that it should be encouraged but not required.

another wording related note I thought of over the weekend; some people may already use something that is not `main` but is also not master as their main branch name (trunk, development, etc.) which accomplishes a lot of what we are trying to do here (we do lose the benefit of consistency across repos, but IMO that's less important)

Comment (tomerd) [reply process/incubation.md]: imo recommending is a sufficiently strong guidelines in the context of incubation process. on the SSWG meeting we also discussed proposing a change to Swift's code of conduct that rejects such language more strongly beyond just branch name

orthogonally, we decided to change all repositories on swift-server org to use `main`

Comment (kmahar) [reply process/incubation.md]: propose we update to this to keep it a recommendation but also address @ktoso's concern
```suggestion
    * Prefer to use `main` as the default branch name, in line with [Swift's guidelines](https://forums.swift.org/t/moving-default-branch-to-main/38515)
```

Comment (ktoso) [reply process/incubation.md]: Sounds good :) Thanks!

**Review by tanner0101 — APPROVED** [review]

**Review by weissi — APPROVED** [review]

**Review by tachyonics — APPROVED** [review]

**Pull request merged**